outputs.tf

```diff
@@ -157,16 +157,6 @@ output "custom_resolver_hub" {
   value       = length(ibm_dns_custom_resolver.custom_resolver_hub) == 1 ? ibm_dns_custom_resolver.custom_resolver_hub[0] : null
 }
 
-output "dns_endpoint_gateways_by_id" {
```

Comment:
Hi Rajat, it's going to break consumers, isn't it? How about returning these outputs using a data ibm_is_vpc_dns_resolution_binding/s lookup in the new version, have you considered this?